projects/dotcms-js/src/lib/core/core-web.service.ts

```diff
@@ -11,7 +11,6 @@ import { Subject, Observable, throwError } from 'rxjs';
 import { map, catchError } from 'rxjs/operators';
 
 import {
-    hasContent,
```

Comment:
Does this have something to do with this PR?

Reply:
not really, but it's causing a warning when running the project